### ISSUE TITLE
Test that ingesting strange characters doesn't break the system

### ISSUE THREAD

**Issue by colincadams** [Recidiviz/recidiviz-data#343]
from #335, we want to make sure that we encode and decode text properly throughout the system. We should wait until after #176 to explore this further.

**Comment by arian487** [comment]
What is the status of this?

**Comment by colincadams** [comment]
I don't believe anyone has looked into this further. It is currently unassigned.
